CHANGELOG.next.asciidoc

```diff
@@ -104,6 +104,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Fix parsing issues with nested JSON payloads in Elasticsearch audit log fileset. {pull}22975[22975]
 - Rename `network.direction` values in crowdstrike/falcon to `ingress`/`egress`. {pull}23041[23041]
 - Change logging in logs input to structure logging. Some log message formats have changed. {pull}25299[25299]
+- Enhance GCP module to populate orchestrator.* fields for GKE / K8S logs {pull}25368[25368]
```

Comment:
Ah, I just noticed that it is under `Breaking changes` section. Shouldn't this go under `Added` section though?

Reply:
Yes correct, done in https://github.com/elastic/beats/pull/25368/commits/09e5aa0c041a5ba6a008b589694f0599c432576e